core/src/test/scala/io/snappydata/CommandLineToolsSuite.scala

```diff
@@ -181,4 +178,99 @@ class CommandLineToolsSuite extends SnappyTestRunner {
       executeCommand(s"rm -rf $snappyHome/backup*")
     }
   }
+
+  test("-dir option with old locator launch script") {
+    try {
+      var consoleOutput = (snappyProductDir +
+          "/sbin/snappy-locator.sh start -peer-discovery-port=10443").!!
+      assert(consoleOutput.contains("ERROR"),
+        s"Option -dir not specified: $consoleOutput")
+
+      consoleOutput = (snappyProductDir +
+          "/sbin/snappy-locator.sh start -peer-discovery-port=10443 -client-port=2000 -dir=").!!
+      assert(consoleOutput.contains("ERROR"),
+        "Option -dir not specified with a value")
+
+      consoleOutput = (snappyProductDir +
+          "/sbin/snappy-locator.sh start -peer-discovery-port=10443 -client-port=2000" +
+          "-dir=/does/not/exist").!!
+      assert(consoleOutput.contains("ERROR"),
+        s"Option -dir does not exist $consoleOutput")
+
+      ("mkdir ./SNAP-2631-work-locator").!!
+      consoleOutput = (snappyProductDir +
+          "/sbin/snappy-locator.sh start -peer-discovery-port=10443 -client-port=2000 " +
+          "-dir=./SNAP-2631-work-locator").!!
+      assert(consoleOutput.contains("running"), s"Locator launch failed: $consoleOutput")
+      consoleOutput = (snappyProductDir +
+          "/sbin/snappy-locator.sh stop -dir=./SNAP-2631-work-locator").!!
+    } finally {
+      (snappyProductDir + "/sbin/snappy-locator.sh stop ").!!
+      ("rm -r ./SNAP-2631-work-locator").!!
+    }
+  }
+  test("-dir option with old server launch script") {
+    try {
+      var consoleOutput = (snappyProductDir +
+          "/sbin/snappy-server.sh start -locators=localhost:10334 -client-port=2001").!!
+      assert(consoleOutput.contains("ERROR"),
+        s"Option -dir not specified: $consoleOutput")
+
+      consoleOutput = (snappyProductDir +
+          "/sbin/snappy-server.sh start -locators=localhost:10334 -client-port=2001 -dir=").!!
+      assert(consoleOutput.contains("ERROR"),
+        "Option -dir not specified with a value")
+
+      consoleOutput = (snappyProductDir +
+          "/sbin/snappy-server.sh start -locators=localhost:10334 -client-port=2001" +
+          "-dir=/does/not/exist").!!
```

Comment:
Missing space between '2001' and '-dir=...'.

Reply:
Added the space